src/cdk/menu/menu-trigger.spec.ts

```diff
@@ -469,6 +470,59 @@ describe('MenuTrigger', () => {
 
     expect(document.querySelector('.test-menu')?.textContent).toBe('Hello!');
   });
+
+  describe('null triggerFor', () => {
+    let fixture: ComponentFixture<TriggerWithNullValue>;
+
+    let nativeTrigger: HTMLElement;
+    let contextNativeTrigger: HTMLElement;
+
+    const grabElementsForTesting = () => {
+      nativeTrigger = fixture.componentInstance.nativeTrigger.nativeElement;
+    };
+
+    /** run change detection and, extract and set the rendered elements */
+    const detectChanges = () => {
+      fixture.detectChanges();
+      grabElementsForTesting();
```

Comment:
I don't think that you need to call this after each change detection. The element is going to stay the same after `ngAfterContentInit`.